Documentation/guides/building-apps/build-items.md

```diff
@@ -213,6 +240,49 @@ step).
 Starting in Xamarin.Android 5.1, attempting to use the `@(Content)`
 Build action will result in a `XA0101` warning.
 
+## EmbeddedJar
+
+In a Xamarin.Android binding project, the **EmbeddedJar** build action
+binds the Java/Kotlin library and embeds the `.jar` file into the
+library. When a Xamarin.Android application project consumes the
+library, it will have access to the Java/Kotlin APIs from C# as well
+as include the Java/Kotlin code in the final Android application.
+
+Since Xamarin.Android 11.2, you can use the **AndroidLibrary**
+build action as an alternative.
+
+## EmbeddedNativeLibrary
+
+In a Xamarin.Android class library or Java binding project, this
+bundles a native library such as `lib/armeabi-v7a/libfoo.so` into the
+library. When a Xamarin.Android application consumes the library, the
+`libfoo.so` file will be included in the final Android application.
+
+Since Xamarin.Android 11.2, you can use the **AndroidNativeLibrary**
+build action as an alternative.
+
+## EmbeddedReferenceJar
+
+In a Xamarin.Android binding project, the **EmbeddedReferenceJar**
+build action embeds the `.jar` file into the library but does not
+create a binding as **EmbeddedJar** does. When a Xamarin.Android
+application project consumes the library, it will include the
+Java/Kotlin code in the final Android application.
+
+Since Xamarin.Android 11.2, you can use the **AndroidLibrary**
+build action as an alternative setting `Bind="false"`:
```

Comment:
Not sure that this is readily interpreted as referring to the *item metadata* `%(Bind)`.  Could this be reworded to make that more explicit?  Or should we use MSBuild item metadata syntax `%(Bind)="false"`?

Reply:
I'll change this to have a short XML snippet like `<AndroidLibrary Include="..." Bind="false" />`

I don't think our user-facing docs should require MSBuild knowledge to know what `%(Bind)` means.